src/main/java/org/photonvision/SimPhotonCamera.java

```diff
@@ -0,0 +1,91 @@
+package org.photonvision;
+
+import java.util.LinkedList;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.wpilibj.geometry.Transform2d;
+
+public class SimPhotonCamera extends PhotonCamera {
+    
+    private Packet packet = new Packet(1);
+
+    LinkedList<PhotonTrackedTarget> trackedTargetList = new LinkedList<PhotonTrackedTarget>();

```

Comment:
When is this list ever cleared?

Reply:
should be at the end of the `update()` method

That being said - per discord conversation - I think we can circumvent this whole thing by going to vararg list of targets which get passed straight to `update()` and not need the list anymore?